.babelrc-cjs.js

```diff
@@ -0,0 +1,9 @@
+const baseConfig = require('./.babelrc');
+
+const config = { ...baseConfig };
+const presetEnvConfig = config.presets.find(
+  ([name]) => name === '@babel/preset-env'
+)[1];
```

Comment:
This lookup is a little gross, but seems necessitated by babel config structure 🤷🏻 . It _seems_ more fragile than it is :). I'd be curious as to the archaeology of the babel-config data structure format...